client/src/components/browse/filterBar/FilterBar.js

```diff
@@ -28,7 +28,9 @@ const FilterBar = ({ itemData, setSelectedItem, setFilteredItems,findNearest, se
   }, [itemData]);
 
   const filterItems = () => {
-    let filteredItems = itemData;
+    let filteredItems = itemData.filter(item => {
+      return !(['false', 'denied'].includes(charity_state) && item.charity_only)
+    });
     if (document.getElementById("distance").value !== "--Select Distance--") {
       filteredItems = filteredItems.filter((item) => {
         return item.distance <= document.getElementById("distance").value;
```

Comment:
@MarkAlperin I wonder if you can use a useRef hook rather the doc.getElementById(), I dont think its anything critical but from my understanding It can help avoid bugs. But honestly Im not too well versed so Id like to hear your thoughts.